pkgs/applications/misc/tuckr/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "tuckr";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "RaphGL";
+    repo = pname;
+    rev = "${version}";
+    sha256 = "sha256-mpI0iAGMIzGGdObH5bfyA3iioNdquzLDZoSWxbAOsJ0=";
+  };
+
+  cargoPatches = [ ./fix-build.patch ]; # Cargo.lock is not up-to-date
```

Use more descriptive names to your patches:

```suggestion
  cargoPatches = [ ./update-cargo-lock.diff ];
```

Also, by convention, `.diff` is the raw difference file, whereas `.patch` is a diff with mail header commentaries.